CHANGELOG.md

```diff
@@ -28,6 +28,9 @@ These changes are available on the `master` branch, but have not yet been releas
 
 - Changed `EmbeddedActivity` values to update accordingly with the new activities.
   ([#1859](https://github.com/Pycord-Development/pycord/pull/1859))
+- Changed the returned value, docstring and fixed a bug where types other than
+  `io.BufferedReader` in `Page.update_files` were throwing `AttributeError`.
+  ([#1869](https://github.com/Pycord-Development/pycord/pull/1869))
```

Comment:
We don't need to list documentation changes, and the typehint updates are stylistic - the new value is equivalent to the old one - so we don't need to list it either. That just leaves the fix, so we can put it under the "fixed" section (see https://keepachangelog.com/en/1.0.0/#types).
```suggestion

### Fixed
- Fixed a bug in `Page.update_files` where file objects stored in memory were causing an `AttributeError`. ([#1869](https://github.com/Pycord-Development/pycord/pull/1869))
```

Reply:
fine, thanks!